docs/wiki/update-alz-custom-policies-to-latest.md

```diff
@@ -0,0 +1,280 @@
+
+## Detect updates to policy
+
+1. To determine if there has been updates to ALZ your first reference should be [What's New](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new). Any updates to policies or other ALZ related artifacts will be reflected here upon release. An example of what that will look like can be seen [here](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new#policy).
+
+2. An alternatively or supplementary tool is [Azure Governance Visualizer](https://github.com/JulianHayward/Azure-MG-Sub-Governance-Reporting) which can be run in your environment and reveal information about the current state of policies and policy assignments. Part of the output of Azure Governance Visualizer is Azure Landing Zones (ALZ) Policy Version Checker which will allows you to see all **outDated** ALZ policies in your environment (see figure 1).
+![AzGovViz-ALZ-Policy-Checker](media/AzGovViz-ALZ-Policy-outDated.png)
+*Figure 1: Azure Governance Visualizer filtering on outDated ALZ policies*
+
+> Note that Azure Governance Visualizer requires permissions in your tenant as described [here](https://github.com/JulianHayward/Azure-MG-Sub-Governance-Reporting#permissions-overview)
+
+## Updating scenarios
+
+These are the following scenarios for ALZ custom policies being updated to latest versions of the custom ALZ policies, listed in increasing order of complexity:
+
+1. One or more ALZ custom policies, whether assigned or not at one or more scopes in your Azure estate, is outdated by a newer ALZ custom policy. The process for managing this is described in [Updating a one or more ALZ custom policy to newer ALZ custom policies](#updating-a-one-or-more-alz-custom-policies-to-newer-alz-custom-policy).
+
+2. One or more ALZ custom policies, assigned at one or more scopes in your Azure estate, is outdated by a newer ALZ custom policy with **updated parameters**. The process for managing this is described in [Updating a one or more ALZ custom policies to a newer ALZ custom policy with updated parameters](#updating-one-or-more-alz-custom-policies-to-newer-alz-custom-policy-with-updated-parameters)
+
+3. One or more ALZ custom policies, assigned via ALZ custom policy initiative, are outdated by newer ALZ custom policy(s) with **updated parameters**. The process for managing this is described in [Updating ALZ custom policies in ALZ custom policy initiative to newer ALZ custom policies](need a reference)
+
+### Updating a one or more ALZ custom policies to newer ALZ custom policy
+
+For this scenario we will use the ALZ custom policy *Deploy Diagnostic Settings for WVD Host Pools to Log Analytics workspace*.
+
+Considering no parameters have changed, this is a simple exercise that consists of replacing the policy definition content with the latest policy definition. While it is possible to update the policy via the portal GUI, there will be a lot of copying and pasting content. To minimize errors, we will be updating this policy via a PowerShell script.
```

Comment:
It's not actually true that updating policy via the portal GUI is a vital option, while most properties can be modified, version cannot via portal GUI at least, so this needs to be rewritten.